### PR TITLE
Duplicate fields from system.process to process.*

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to rename several fields from `system.process` metricset to `process.*`. Here is the full list:

- system.process.state -> process.state
- system.process.cpu.total.norm.pct -> process.cpu.pct
- system.process.cpu.start_time -> process.cpu.start_time
- system.process.memory.rss.pct -> process.memory.pct

cc @fearful-symmetry @mukeshelastic 

## Why is it important?

This is a part of the inventory schema work for process. Please see https://github.com/elastic/beats/issues/22216 for more details.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have made corresponding change to the default configuration files~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

**Comment by andresrc** [comment]
should we do this in the package as well?

**Comment by kaiyan-sheng** [comment]
@andresrc Do you mean adding the new fields into the system integration? I'm planning to create a PR to move it into the integration after this PR is merged.

File: metricbeat/module/system/process/process.go [hunk 1]
```diff
@@ -143,6 +143,32 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
...
+		// Duplicate system.process.state with process.state
+		if v, ok := proc["state"]; ok {
+			rootFields.Put("process.state", v)
+		}
+
+		// Duplicate system.process.cpu.start_time with process.cpu.start_time
+		// Duplicate system.process.cpu.total.norm.pct with process.cpu.pct
+		if k, ok := proc["cpu"]; ok {
+			cpu := k.(common.MapStr)
+			if v, ok := cpu["start_time"]; ok {
```
Comment (fearful-symmetry) [inline metricbeat/module/system/process/process.go:155]: I wonder if it makes sense to make a `getAndCopy` function here like `getAndRemove` to cut down on some of this `if`-block cruft, potentially using the `CopyFieldsTo` method for MapStr.

Comment (kaiyan-sheng) [reply metricbeat/module/system/process/process.go:155]: Perfect! Will do!

**Review by fearful-symmetry — APPROVED** [review]
LGTM

**Pull request merged**